src/mlpack/tests/CMakeLists.txt

```diff
@@ -110,7 +111,6 @@ add_executable(mlpack_test
   timer_test.cpp
   tree_test.cpp
   tree_traits_test.cpp
-  scaling_test.cpp
```

Comment:
Any reason to remove this test?  Or did I miss something?

Reply:
Nice catch! Looks like an incorrect merge.

Reply:
Fixed.

Reply:
Hmm, maybe I overlooked something but it seems like this test is still removed in the diff.